source/_integrations/apple_tv.markdown

```diff
@@ -190,3 +258,17 @@ data:
     - menu
     - select
 ```
+
+## Debugging
+
+If you have any problems and intend to write an issue, make sure you have the
+relevant logs included. For this component, you can enable them like this:
+
+```yaml
+logger:
+  logs:
+    pyatv: debug
+    homeassistant.components.apple_tv: debug
+```
+
+Without logs you will most likely not get any help.
```

Comment:
Let's make this a bit friendlier 😉 
```suggestion
Supplying these logs helps us tremendously resolving an issue.
```

Reply:
I actually thought about it when writing this, but I went with the bad ass attitude because of reasons 😂 I will make it a bit nicer.